gradle.properties

```diff
@@ -19,4 +19,4 @@ org.gradle.daemon=true
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
```

Comment:
this is not a change. then why is it here. can you rectify that

Reply:
Reverted

Reply:
it is not yet reverted.